src/cdomains/baseDomain.ml

```diff
@@ -115,7 +115,7 @@ struct
   let printXml f r =
     let e = XmlUtil.escape in
     BatPrintf.fprintf f "<value>\n<map>\n<key>\n%s\n</key>\n%a<key>\n%s\n</key>\n%a<key>\n%s\n</key>\n%a\n<key>\n%s\n</key>\n%a</map>\n</value>\n"
-      (e @@ CPA.name ()) CPA.printXml r.cpa
+      (e @@ (CPA.name ())) CPA.printXml r.cpa
```

Comment:
There's a number of irrelevant changes like this due to the addition and then removal of `RenameMapping` I guess. If you look at the entire PR's changes on GitHub you can easily find where unnecessary extra parenthesis are added or whitespace is changed in otherwise unchanged code. Would be nice to undo these changes such that this PR doesn't introduce noise to `git blame`.